login/service_blackbox_test.go

```diff
@@ -50,8 +43,6 @@ func TestRunServiceBlackBoxTest(t *testing.T) {
 // It sets up a database connection for all the tests in this suite without polluting global space.
 func (s *serviceBlackBoxTest) SetupSuite() {
 	s.DBTestSuite.SetupSuite()
-	s.ctx = migration.NewMigrationContext(context.Background())
-	s.DBTestSuite.PopulateDBTestSuite(s.ctx)
 
 	var err error
 	s.configuration, err = config.GetConfigurationData()
```

Comment:
this could also be replaced with `s.Configuration`

Reply:
Why is that?

Reply:
because `s.DBTestSuite` already contains a `Configuration` field, doesn't it ?

Reply:
Ah, that's what you mean. I was wondering why on earth you wanted me to make `s.configuration` public. :) Fixed in 06037339

Reply:
ah ok, sorry for the misunderstanding ;) yeah, let's just use the public, inherited `s.Configuration` ;)